.github/workflows/static-root-checks.yml

```diff
@@ -29,3 +29,5 @@ jobs:
         run: yarn tsc --noEmit
       - name: Lint
         run: yarn lint:js-root
+      - name: Check for circular dependencies
+        run: yarn madge --extensions js,ts,tsx --circular src
```

Comment:
Would it be possible to add script in our `package.json` and then use it here? Like `yarn circular-dependency-check`. This way it would be easier to run this in our repo (we wouldn't have to remember all those arguments and so on).

Reply:
ok, will do

Reply:
don't we want a script for both the JS and the native side in this case? 
or is this a bit far fetched for now?

Reply:
Is `madge` capable of doing such things?

Also keep in mind that in order to add this script we would have to have 0 circular dependencies on native side (I don't say we have them, that's something that we have to check).

So in conclusion, it would be great to have such script for native, but not in this PR.

Reply:
Although `madge` likely won't work, for `kotlin` there is a tool called `Classycle` which could be made into a workflow.

But of course, i don't think doing this in this PR would be a good idea either 😄 
I was just asking if it was something you'd like to be done.

Reply:
I think it could be done, though first, it would be great to check how many of them are in the codebase to have an idea of the effort it would take.